src/PerfView/UserCommands.cs

```diff
@@ -1512,6 +1533,36 @@ private TraceEvents GetTraceEventsWithProcessFilter(ETLDataFile etlFile)
             return events;
         }
 
+        /// <summary>
+        /// Save the CPU stacks for an ETL file into a perfView.xml.zip file.
+        /// </summary>
+        /// <param name="etlFile">The ETL file to save.</param>
+        /// <param name="process">The process to save. If null, save all processes.</param>
+        /// <param name="filter">The filter to apply to the stacks. If null, apply no filter.</param>
+        private static void SaveCPUStacksForProcessAsCsv(ETLDataFile etlFile, TraceProcess process = null, FilterParams filter = null)
+        {
+            // Focus on a particular process if the user asked for it via command line args.
+            if (process != null)
+            {
+                etlFile.SetFilterProcess(process);
+            }
+
+            if (filter == null)
+            {
+                filter = new FilterParams();
+            }
+
+            var stacks = etlFile.CPUStacks();
+            stacks.Filter = filter;
+            stacks.LookupWarmSymbols(10); // Look up symbols (even on the symbol server) for modules with more than 10 inclusive samples
```

Comment:
Is it worth adding the inclusive sample threshold as a parameter with a default value of 10?

Reply:
Probably!  This is what the previous one was doing but who says we cant iterate on the changes and make it better and more modular :)